decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/participatory_process_copies_controller.rb

```diff
@@ -15,9 +15,9 @@ def new
 
         def create
           enforce_permission_to :create, :process
-          @form = form(ParticipatoryProcessCopyForm).from_params(params)
+          @form = form(ParticipatoryProcessCopyForm).from_params(params).with_context(current_organization:)
```

Comment:
I think this should be current_user instead of current_organization (or it isn't even necessary as the form seems like it already has it)
```suggestion
          @form = form(ParticipatoryProcessCopyForm).from_params(params).with_context(current_user:)
```

Reply:
I remember I altered this change because it broke initially with the same failed output. 

I will revert this commit to its original state, unless you have another suggestion @andreslucena

Reply:
By doing a quick check, I have another suggestion: we don't need any extra context for this form, at least by the specs runs that I'm running: 

```suggestion
          @form = form(ParticipatoryProcessCopyForm).from_params(params)
```